samples/appending-events/appending-events.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <RootNamespace>appending_events</RootNamespace>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\..\src\EventStore.Client.Streams\EventStore.Client.Streams.csproj" />
```

Comment:
Should we reference the nuget packages instead? This might help alleviate the confusion around which packages to install. Alternatively, we could include the nuget packages but comment them out.

Reply:
@thefringeninja my initial idea was to take those snippets as they're. My next task is to prepare more real-world samples. I planned then to revisit also those snippets and have new samples with NuGet packages. Thoughts?

Reply:
Added commented package reference.